conda/build_pytorch.sh

```diff
@@ -321,7 +321,7 @@ for py_ver in "${DESIRED_PYTHON[@]}"; do
 
     # We need to build the compiler activation scripts first on Windows
     if [[ "$OSTYPE" == "msys" ]]; then
-        vs_package="vs$VC_YEAR"
+        vs_package="vs$VS_YEAR"
```

Comment:
I am not sure about this. There is/was a discrepancy between `VS_YEAR` and `VC_YEAR` variable naming. From Conda standpoint, `VC_YEAR` does not exists vile `VS_YEAR` does (https://docs.conda.io/projects/conda-build/en/stable/user-guide/environment-variables.html).